dashboards-observability/common/constants/explorer.ts

```diff
@@ -80,3 +80,6 @@ export const PLOTLY_GAUGE_COLUMN_NUMBER = 5;
 export const APP_ANALYTICS_TAB_ID_REGEX = /application-analytics-tab.+/;
 export const DEFAULT_AVAILABILITY_QUERY = 'stats count() by span( timestamp, 1h )';
 export const PPL_PATTERNS_REGEX = /\|\s*patterns\s+\S+\s*\|\s*where\s+patterns_field\s*\=\s*'[^a-zA-Z0-9]+'/;
+export const PATTERNS_REGEX = /\|\s*patterns\s+\S+\s*\|.*\s*where\s+patterns_field\s*\=\s*'[^a-zA-Z0-9]+'/;
```

Comment:
what's the difference between PPL_PATTERNS_REGEX PATTERNS_REGEX and SELECTED_PATTERN_REGEX? could you add some comments since they look similar

Reply:
@joshuali925 added some comments - let me know if tis comprehensive/you understand what each one is for